tensorflow/lite/toco/graph_transformations/hardcode_min_max.cc

```diff
@@ -279,7 +279,7 @@ bool MinMaxApproximatelyEqual(const MinMax& minmax1, const MinMax& minmax2) {
 // If multiple of these arrays have MinMax, then these are required
 // to agree with each other.
 bool PropagateMinMaxAmongArrays(Model* model,
-                                const std::vector<string> array_names) {
+                                const std::vector<string> &array_names) {
```

Comment:
Nit: the `&` should go by the type rather than the arg name:

const std::vector<string>& array_names

Reply:
@jdduke , thank you for the review. However I do not understand the point. Do you mean replace: std::vector<string> with std::vector ? 

For the vector template class, you have to define the type everywhere, (unless you are using auto)

The following sample code in C++ works:
#include <iostream>
#include <vector>

using namespace std;

void pass_by_ref(const vector<string> &v)
{
//do nothing
return;
}

int main()
{
vector<string> v = {"hello", "world"};
pass_by_ref(v);
return 0;
}

If I change the signature of the function to: void pass_by_ref(const vector &v), it gives me a compilation error.

Thoughts ?

Reply:
No, I simply mean move the `&` one space to the left from your original change, so it's next to the type, not the arg name.